tests/unit/test_batch_rewards_schema.py

```diff
@@ -24,13 +24,13 @@ def test_order_rewards_transformation(self):
                 "execution_cost": [9999 * ONE_ETH, 1],
                 "surplus": [2 * ONE_ETH, 3 * ONE_ETH],
                 "fee": [
-                    0.01 * ONE_ETH,
-                    0.0,
+                    1000000000000000,
+                    0,
                 ],
-                "uncapped_payment_eth": [0.0, -10 * ONE_ETH],
-                "capped_payment": [-0.001 * ONE_ETH, -0.001 * ONE_ETH],
-                "winning_score": [11761680942929144.0, 11296611208025016.0],
-                "reference_score": [11761680942929126.0, 11296611208025016.0],
+                "uncapped_payment_eth": [0, -10 * ONE_ETH],
+                "capped_payment": [-1000000000000000, -1000000000000000],
+                "winning_score": [123456 * ONE_ETH, 6789 * ONE_ETH],
+                "reference_score": [ONE_ETH, 2 * ONE_ETH],
```

Comment:
Just a nit, not really important: those values are not consistent with the reward scheme. Since this is just for testing some method it should not matter, though.
[E.g. capped payment cannot be negative when uncapped payment is non-negative; uncapped payment cannot be negative and larger than reference score in absolute value.]

Reply:
You are welcome to suggest whatever numbers you like here although it really has no impact since its just a test that the parsing and constructor work as expected (as you mentioned).